fl4health/clients/basic_client.py

```diff
@@ -273,6 +273,20 @@ def fit(self, parameters: NDArrays, config: Config) -> Tuple[NDArrays, int, Dict
         # Update after train round (Used by Scaffold and DP-Scaffold Client to update control variates)
         self.update_after_train(local_steps, loss_dict)
 
+        # Check if we should run an evaluation with validation data after fit (used by FedDGGA)
+        try:
+            evaluate_after_fit = self.narrow_config_type(config, "evaluate_after_fit", bool)
+        except ValueError:
+            evaluate_after_fit = False
```

Comment:
Rather than doing this try catch here, we can move this functionality into `process_config`?

Reply:
Moved, please check.